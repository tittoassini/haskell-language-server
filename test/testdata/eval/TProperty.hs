-- Support for property checking
module TProperty where

-- prop> \(l::[Bool]) -> reverse (reverse l) == l

-- prop> \(l::[Bool]) -> reverse l == l

