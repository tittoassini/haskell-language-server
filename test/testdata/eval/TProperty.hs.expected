-- Support for property checking
module TProperty where

-- prop> \(l::[Bool]) -> reverse (reverse l) == l
-- +++ OK, passed 100 tests.

-- prop> \(l::[Bool]) -> reverse l == l
-- +++ OK, passed 100 tests.

